<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWindow</class>
 <widget class="QWidget" name="UpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑员工信息</string>
  </property>
  <widget class="QLabel" name="idLab">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>市场部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>研发部</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>运维部</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
